README.md

```diff
@@ -10,7 +10,7 @@ Install the dependencies
 
 ```
 npm install @xata.io/client
-npm install @xata.io/codegen -D
+npm install @xata.io/cli -g
```

Comment:
I was expecting to see `-D` still for this? Since if a teammate checks out your project it wouldn't work with `-g`.